.gitignore

```diff
@@ -1,4 +1,7 @@
-# Ignore directories generated by Composer
+# To ignore OS temporary files use global .gitignore
```

Comment:
The local-specific overrides should be managed by users themselves using a global `.gitignore`. Ignoring vendor-specific entries make this selection biased, so it is better to exclude them from the list. The entries removed below.